runelite-client/src/main/java/net/runelite/client/plugins/bankvalue/BankValuePlugin.java

```diff
@@ -78,12 +81,28 @@ public void onGameTick(GameTick event)
 		}
 	}
 
+	@Subscribe
+	public void onConfigChanged(ConfigChanged event)
+	{
+		if (bankCalculation.getGePrice() > 0)
+		{
+			Widget widgetBankTitleBar = client.getWidget(WidgetInfo.BANK_TITLE_BAR);
+
+			if (widgetBankTitleBar == null || widgetBankTitleBar.isHidden())
+			{
+				return;
+			}
+
+			bankTitle.update(bankCalculation.getGePrice(), bankCalculation.getHaPrice(), true);
+		}
+	}
+
 	private void calculate(Widget bankTitleBar)
 	{
 		// Don't update on a search because rs seems to constantly update the title
 		if (bankTitleBar == null ||
-			bankTitleBar.isHidden() ||
-			bankTitleBar.getText().contains("Showing"))
+				bankTitleBar.isHidden() ||
+				bankTitleBar.getText().contains("Showing"))
```

Comment:
Why was the indent on this changed? As mentioned above you should probably review the following article:
https://github.com/runelite/runelite/wiki/Checkstyle-with-IntelliJ-IDEA

Reply:
Format with via. import from Checkstyle.

Importing the checkstyle contained in this project yields the above indentations. What do I need to change in my format settings to avoid this? I am an avid user of auto formatting.